### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by SashaShostyr**

### CODE REVIEW

**Review by idealasgas — APPROVED** [review]

**Review by reedzyy — APPROVED** [review]

**Comment by SashaShostyr** [comment]
**Need code review!**

File: spec/controllers/items_controller_spec.rb [hunk 1]
```diff
@@ -51,7 +103,7 @@
       it "re-renders the new method" do
         post :create, params: { item: invalid_attributes }
         item = assigns(:item)
-        expect(item.errors).to be_present
+        expect(item.errors.messages[:title]).to be_present
```
Comment (anatoliliotych) [inline spec/controllers/items_controller_spec.rb]: what message?

File: spec/controllers/items_controller_spec.rb [hunk 2]
```diff
@@ -60,4 +112,65 @@
...
+      end
+
+      it "does not change item's attributes" do
+        item.reload
+        expect(item.title).to_not be_nil
+      end
+
+      it "re-renders the edit method " do
+        item = assigns(:item)
+        expect(item.errors.messages[:title]).to be_present
```
Comment (anatoliliotych) [inline spec/controllers/items_controller_spec.rb]: what message?

File: spec/controllers/sessions_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Users::SessionsController, type: :controller do
+  login_user
+  let(:user) { subject.current_user }
+
+  controller do
```
Comment (anatoliliotych) [inline spec/controllers/sessions_controller_spec.rb]: could it be moved to some support class? doesn't look like something we need in this file

**Review by anatoliliotych — CHANGES_REQUESTED** [review]

**Review by anatoliliotych — APPROVED** [review]
LGTM

**Pull request merged**